src/scripts/main.js

```diff
@@ -1,3 +1,26 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+const messageDiv = document.createElement('div');
+
+messageDiv.classList.add('message');
+
+function handler() {
+  const promise1 = Promise.resolve('Promise was resolved!');
+
+  promise1.then((result) => {
+    messageDiv.innerText = result;
+    document.body.appendChild(messageDiv);
+  });
+
+  const promise2 = Promise.reject(new Error('Promise was rejected!'));
+
+  promise2.catch((error) => {
+    setTimeout(() => {
+      messageDiv.classList.add('error-message');
+      messageDiv.innerText = error.message;
+    }, 3000);
+  });
+}
```

Comment:
It's always good to have function names that describe what they do. In this case, 'handler' is a bit too generic. Consider renaming it to something more descriptive, like 'handleLogoClick'.